### PR TITLE
api project completed - hopefully!

### PR DESCRIPTION
**Pull request by ShlomoRBarron**

### CODE REVIEW

File: student-work/2023/Shlomo-Bar-On/api-final-project/get.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+|Cost|Float|Cost of burgerMeal
+|Item2|JSON object|The side order menu item ordered.
+|ItemOrdered|Nested JSON object|Detail of Item2
+|type|String|pommeFritesMedium
+|Cost|Float|Cost of side order type
+
+
+---
+
+> Explore other templates from [The Good Docs Project](https://thegooddocsproject.dev/). Use our [feedback form](https://thegooddocsproject.dev/feedback/?template=API%20reference) to give feedback on this template.
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Shlomo-Bar-On/api-final-project/get.md:99]:
```suggestion
> Explore other templates from [The Good Docs Project](https://thegooddocsproject.dev/). Use the [feedback form](https://thegooddocsproject.dev/feedback/?template=API%20reference) to give feedback on this template.
```

File: student-work/2023/Shlomo-Bar-On/api-final-project/post.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+	"cook_id": "l2j23j9LKJsd",
+	"price_base": "10.99", 
+	"price_add": "8.00", 
+	"price_total": "18.99" 
+	}
+}
+```
+---
+
+> Explore other templates from [The Good Docs Project](https://thegooddocsproject.dev/). Use our [feedback form](https://thegooddocsproject.dev/feedback/?template=API%20reference) to give feedback on this template.
```
Comment (Laura-Novich-OBW) [inline student-work/2023/Shlomo-Bar-On/api-final-project/post.md:207]:
```suggestion
> Explore other templates from [The Good Docs Project](https://thegooddocsproject.dev/). Use the [feedback form](https://thegooddocsproject.dev/feedback/?template=API%20reference) to give feedback on this template.
```

**Review by Laura-Novich-OBW — CHANGES_REQUESTED** [review]
one small change to two files, but otherwise amazing!

**Comment by ShlomoRBarron** [comment]
Changes committed.

Ready for merge.

**Review by Laura-Novich-OBW — APPROVED** [review]

**Comment by Laura-Novich-OBW** [comment]
nice job please submit to Google Classroom

**Pull request merged**